packages/transformer/src/IModelTransformer.ts

```diff
@@ -2346,7 +2360,9 @@ export class IModelTransformer extends IModelExportHandler {
   // FIXME<MIKE>: is this necessary when manually using low level transform APIs? (document if so)
   private finalizeTransformation() {
     this.importer.finalize();
-    this.updateSynchronizationVersion();
+    this.updateSynchronizationVersion({
+      initializeReverseSyncVersion: this._isProvenanceInitTransform,
```

Comment:
comma not needed

Reply:
Might be prettiers doing? not sure

Reply:
That's correct. Prettier adds these commas
![image](https://github.com/user-attachments/assets/01f2a8aa-848e-48c7-87aa-95464bc738ed)